x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPolicyRunner.java

```diff
@@ -294,15 +390,17 @@ private void prepareAndCreateEnrichIndex() {
             // This disables eager global ordinals loading for all fields:
             .put("index.warmer.enabled", false)
             .build();
-        CreateIndexRequest createEnrichIndexRequest = new CreateIndexRequest(enrichIndexName, enrichIndexSettings);
```

Comment:
I think this can be restored? I think this `resolveEnrichMapping(...)` and `resolveRangeTypeEnrichMapping(...)` can now just return the mapping via the return value? (since `resolveRangeTypeEnrichMapping(...)` doesn't invoke the get field mapping api any more).

Reply:
addressed in https://github.com/elastic/elasticsearch/pull/76110/commits/2924d2bb76c3cd027a1bf129f2de39b08a2461aa